dvc/params.yaml

```diff
@@ -1,44 +1,3 @@
-datasets:
-  in:
-    dataset_name: interneurons
-    id: in
-    input_dataset_directory: data/raw/interneurons
-    intermediary_directory: reports/data_preparation/interneurons
-    output_dataset_directory: data/prepared/interneurons
-    output_wildcard: '''data/prepared/interneurons/*/*/*.h5'''
-  pc:
-    dataset_name: pyramidal_cells
-    id: pc
-    input_dataset_directory: data/raw/pyramidal_cells
-    intermediary_directory: reports/data_preparation/pyramidal_cells
-    output_dataset_directory: data/prepared/pyramidal_cells
-    output_wildcard: '''data/prepared/pyramidal_cells/*/*/*.h5'''
-explain_models:
```

Comment:
Seems fine to remove, `explain-models` DVC stage was now removed by `xai` DVC stage.